### PR TITLE
[PRED-1532] set doublequote to True, if escapechar is not detected

### PR DESCRIPTION
**Pull request by sevikkk**
We need one or another, otherwise writer can't write values with
quotes in them.

### CODE REVIEW

File: datarobot_batch_scoring/reader.py [hunk 1]
```diff
@@ -406,6 +406,11 @@ def sniff_dialect(sample, encoding, sep, skip_dialect, ui):
         dialect = sniffer.sniff(resampled, delimiters=delimiter)
         ui.debug('investigate_encoding_and_dialect v2 - seconds to detect '
                  'csv dialect: {}'.format(time() - t2))
+
+    if dialect.escapechar is None:
+        csv.register_dialect('dataset_dialect', dialect,
+                             doublequote=True)
+        dialect = csv.get_dialect('dataset_dialect')
```
Comment (dsakagi) [inline datarobot_batch_scoring/reader.py]: Could you add a test of `sniff_dialect` that shows this behaving as you expect?

**Comment by sevikkk** [comment]
the problem is - we need pretty big dataset for this - as we need dataset without quoted data in first 10 megs, and with quotes after it..

Ok, I can generate it dynamically

**Comment by dsakagi** [comment]
Seva: Master Problem Solver :crown:

**Comment by dsakagi** [comment]
Well I guess that's one way to do it :)

I probably would have generated it all dynamically instead of having those three test fixtures, but I'm not opposed to shipping it like this when it goes green.

File: datarobot_batch_scoring/reader.py (hunk 1 above)
Comment (dsakagi) [reply datarobot_batch_scoring/reader.py]: n.b. it has been added

**Pull request merged**